src/App.js

```diff
@@ -97,8 +97,8 @@ const App = ({ locale, location, dependencies: { appSessionRef, sessionManager }
           <PrivateRoute path="/reports/" exact requireSiteTracking component={Reports} />
           <PrivateRoute path="/integrations/shopify" exact component={Shopify} />
           <PrivateRoute path="/reports/master-subscriber" exact component={MasterSubscriber} />
-          <PrivateRoute path="/reports/subscriber-history" exact component={SubscriberHistory} />
-          <PrivateRoute path="/reports/subscriber-gdpr" exact component={SubscriberGdpr} />
+          <PrivateRoute path="/subscribers/:email/:section" component={Subscribers} />
+          <PrivateRoute path="/reports/:oldParam" exact component={SubscribersRedirect} />
```

Comment:
Is there a way to accept only the segments that we are interested in? 

Pseudocode:

```suggestion
          <PrivateRoute path="/reports/:oldParam" exact component={SubscribersRedirect} restrictions:{{oldParam: /subscriber-(history)|(gdpr)/i}}/>
```

Reply:
Do you think that this is necessary? I have to delete this when release is done.  By the way, I should add a TODO comment to delete this.

Reply:
Probably not, but it is a good opportunity to learn something useful.